### PR TITLE
 Added telugu-IN laguage

### PR DESCRIPTION
**Pull request by cbaba20**
Telugu is one of widely spoken language in India and a large number of people use firefox in this language.

### CODE REVIEW

File: locales/te-IN [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+hashtag = #newfirefox
+github = గిఠ్హబ్
+legal = చట్టపరమైనవి
+license = లైసెన్సు
+guidelines = మార్గదర్శకాలు
+mozilla-description =
+మొజిల్లా ప్రపంచవ్యాప్త లాభాపేక్షలేనిది, మీ ఆన్ లైన్ అనుభవాన్ని నియంత్రించి, భవిష్యత్ను రూపొందించడంలో అంకితం చేయబడింది
+  ప్రజా మంచి కోసం వెబ్. మమ్మల్ని సందర్శించండి
+  <a href="https://mozilla.org" target="_blank">mozilla.org</a>
+official-tweet-id = 928033132908986369
```
Comment (MichaelKohler) [inline locales/te-IN]: as this is an old tweet and was only used as an example until now, I'm in the process of removing it. Could you please set this string to "tbd"? I will update this tomorrow for all language files once the official tweet is out.

Thanks!

**Review by MichaelKohler — APPROVED** [review]
Thanks, please see my comment.

File: locales/te-IN [hunk 2]
```diff
@@ -0,0 +1,21 @@
+title = ఫైర్ఫాక్స్  క్వాంటం విడుదల ప్రచారం
```
Comment (MichaelKohler) [inline locales/te-IN]: can you please rename the file to te-IN.ftl (see the file format)

**Review by MichaelKohler — CHANGES_REQUESTED** [review]

File: locales/te-IN (hunk 2 above)
Comment (cbaba20) [reply locales/te-IN]: Yes, doing it now.

File: locales/te-IN (hunk 1 above)
Comment (cbaba20) [reply locales/te-IN]: Done ! please check.

**Review by MichaelKohler — APPROVED** [review]
Thank you!

**Pull request merged**